src/devices/oria_wa150km.c

```diff
@@ -0,0 +1,132 @@
+/** @file
+    Oria WA150KM temperature sensor decoder.
+
+    Copyright (C) 2025 Jan Niklaas Wechselberg
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+/*
+The device uses Manchester coding with G.E. Thomas convention.
+The data is bit-reflected.
+
+Data layout after decoding:
+     0  1  2  3  4  5  6  7  8  9  10 11 12 13
+[00] FF FF FF ?? ?? CC DD TT II SS ?? ?? ?? BB
```

Comment:
Indent this line with 4 spaces, e.g. remove `[00]`. A blank line above and below this indented block.